src/Widgets/TaskRow.vala

```diff
@@ -113,20 +127,27 @@ public class Tasks.TaskRow : Gtk.ListBoxRow {
                 due_datetime_popover.get_style_context ().add_class ("error");
             }
 
-            var h24_settings = new GLib.Settings ("org.gnome.desktop.interface");
-            var format = h24_settings.get_string ("clock-format");
-
-            if (is_scheduled_view) {
-                return _("%s").printf (
-                    value.format (Granite.DateTime.get_default_time_format (format.contains ("12h")))
-                );
-
+            if (is_gtask) {
+                if (is_scheduled_view) {
+                    return null;
```

Comment:
Not 100% sure if it is related to this, but I can't see any GTasks in my Scheduled view. Is this supposed to work already?

Reply:
Since we can't check for due on the s-expression (#149), they don't appear even on master, I only added this for completeness.

Reply:
Agree. That's why we set start date to the same value as the due date (https://github.com/elementary/tasks/pull/147). Isn't this possible using GTasks?

Reply:
Since GTasks tasks are GData.TasksTask instead of Ical.Component, EDS needs to convert them, and only sets the due date on the conversion. So they do not have a start time when they come from the server

Reply:
Alright, thanks for the clarification! I made an issue to keep track of this: https://github.com/elementary/tasks/issues/205